internal/venice-avro-compatibility-test/src/test/java/com/linkedin/venice/VeniceClusterInitializer.java

```diff
@@ -79,7 +79,7 @@ public class VeniceClusterInitializer implements Closeable {
 
   public VeniceClusterInitializer(String storeName, int routerPort) {
     Properties clusterConfig = new Properties();
-    clusterConfig.put(ConfigKeys.SERVER_PROMOTION_TO_LEADER_REPLICA_DELAY_SECONDS, 1L);
+    clusterConfig.put(ConfigKeys.SERVER_PROMOTION_TO_LEADER_REPLICA_DELAY_SECONDS, 5L);
```

Comment:
Can we have a constant instead? Ideally, if we were to change this once again, it should be a one line change next time...